source/require-external-modules.txt

```diff
@@ -0,0 +1,177 @@
+=============================================
+Include External Files and Modules in Scripts
+=============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. important::
+
+   A complete description of Node.js, modules, and the
+   `require() <https://nodejs.org/api/modules.html#modules_require_id>`__
+   function is out of scope for this tutorial. To learn more, see
+   the `Node.js Documentation <https://nodejs.org/api/modules.html>`__.
+
+To use files and modules in your ``mongosh`` interactions, use the
+`require() <https://nodejs.org/api/modules.html#modules_require_id>`__
+function.
+
+In your ``mongosh`` scripts, you can require:
+
+- Local files
+- Built-in Node.js modules
+- External (npm) Node.js modules
+
+Require a Local File
+--------------------
+
+You can use JavaScript files in ``mongosh`` scripts without any
+additional setup or configuration.
+
+.. note::
+
+   ``mongosh`` does not execute files imported with ``require()``.
+   Instead, ``mongosh`` adds everything from an imported file to the
+   current execution scope.
+
+.. example::
+
+   To include a file named ``test.js`` that is located in the current
+   working directory, use one of the following commands:
+
+   .. code-block:: javascript
+
+      require('./tests.js')
+
+   .. code-block:: javascript
+      
+      var tests = require('./tests.js')
+
+Require a Built-In Module
+-------------------------
+
+You can require built-in Node.js modules (such as
+`fs <https://nodejs.org/api/fs.html#fs_file_system>`__) in ``mongosh``
+without any additional setup or configuration.
+
+.. example::
+
+   The following example creates and executes a script that:
+   
+   - Connects to a local deployment running on the default port.
+   - Populates the ``myDatabase.employees`` collection with sample data.
+   - Uses the ``fs`` module to write a document from the
+     ``myDatabase.employees`` collection to a file named
+     ``employee.txt``.
+
+   1. Create a file named ``employee-to-text-file.js`` with the
+      following contents:
+
+      .. code-block:: javascript
+
+         const fs = require('fs');
+
+         conn = Mongo();
+         db = conn.getDB("myDatabase");
```

Comment:
Feel free to take it or leave it, but I feel like `connect()` might be more intuitive/more self-explanatory for users who don’t know the `Mongo` constructor:

```suggestion
         db = connect('mongodb://localhost/myDatabase');
```